### ISSUE TITLE
WCAG 4.1.2: Ensures every form element has a label (input[type="text"])

### ISSUE THREAD

**Issue by joosep41** [joosep41/Manguarvustus#4]
#### Issue
`Ensures every form element has a label` ([label](https://accessibilityinsights.io/info-examples/web/label))
#### Target application
[Accessibility Fails](file:///C:/Users/j/dev/accessibility-fails/index.html)
#### Element path
.example:nth-child(201) > input[type="text"]
#### Snippet
`<input type="text">`
#### How to fix
    Fix any of the following:
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element has no placeholder attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Chrome version 110.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
